inspire/modules/workflows/tasks/submission.py

```diff
@@ -256,3 +256,15 @@ def add_files_to_task_results(obj, eng):
         obj.add_task_result(file_obj.name,
                             fileinfo,
                             "workflows/results/files.html")
+
+
+def add_note_entry(obj, eng):
+    """Add metadata to sip on approval."""
+    if was_approved(obj, eng):
+        rt = get_instance() if cfg.get("PRODUCTION_MODE") else None
```

Comment:
Since the task was
```
Core papers:
500__ $$a*Temporary entry*  <--- RT ticket generated for these
```

I understood it as `this has to be core AND has to have a RT ticket` statement.

Are there that many tasks core [exclusive] related to open a new `workflow_if/else` or you're worried about the future? ^^